x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -94,7 +94,7 @@ export interface TGridStandaloneProps {
   loadingText: React.ReactNode;
   filters: Filter[];
   footerText: React.ReactNode;
-  filterStatus: AlertStatus;
+  filterStatus?: AlertStatus;
```

Comment:
All tests passed, but I still want to verify that it is ok to make this prop optional.

Reply:
Ya should be fine, it's only used in one place, passed through to StatefulBody where it's already optional https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/public/components/t_grid/body/index.tsx#L100